net/respondd-module-wifi/src/respondd.c

```diff
@@ -0,0 +1,124 @@
+#include <string.h>
+#include <stdio.h>
+#include <json-c/json.h>
+#include <respondd.h>
+
+#include <linux/nl80211.h>
+
+#include "airtime.h"
+#include "clients.h"
+#include "neighbours.h"
+#include "ifaces.h"
+
+static struct json_object *respondd_provider_statistics(void) {
+	struct json_object *result, *wireless, *clients, *interface;
+	struct iface_list *ifaces;
+	int wifi24 = 0, wifi5 = 0, clients_count;
+
+	result = json_object_new_object();
+	if (!result)
+		return NULL;
+
+	wireless = json_object_new_array();
+	if (!wireless) {
+		//TODO why needed? : json_object_put(result);
+		return NULL;
+	}
+
+	clients = json_object_new_object();
+	if (!clients) {
+		return NULL;
+	}
+
+	ifaces = get_ifaces();
+	while (ifaces != NULL) {
+		clients_count = 0;
+		get_client_counts(&clients_count, ifaces->ifx);
+		if(ifaces->type == NL80211_IFTYPE_AP) {
+			if (ifaces->frequency < 5000)
+				wifi24 += clients_count;
+			if (ifaces->frequency > 5000)
+				wifi5 += clients_count;
+		}
+
+		//TODO wiphy only one radio added? (not necessary on gluon - only one ap-ssid at radio)
```

Comment:
what about private wifi?

Reply:
They are skipped by this line:
https://github.com/freifunk-gluon/packages/pull/188/files#diff-372361d4845238ed37120eb1554c176eR31
Thankyou for your review, i will correct this